src/protobuf-net.AspNetCore/Formatters/ProtoInputFormatter.cs

```diff
@@ -13,14 +15,20 @@ namespace ProtoBuf.Formatters
     public sealed class ProtoInputFormatter : InputFormatter
     {
         private readonly TypeModel _model;
+        private readonly int _memoryBufferThreshold;
+
+        // influened directly by https://github.com/dotnet/aspnetcore/blob/master/src/Mvc/Mvc.NewtonsoftJson/src/MvcNewtonsoftJsonOptions.cs
+        internal const int DefaultMemoryBufferThreshold = 1024 * 30;
```

Comment:
FYI we have a work item to make this bigger, it's wayy too small right now.

Reply:
k, I'll go with something larger and await what number you decide on in the end